keyboards/qneatandtidy/dx65/hotswap/rev1/config.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2023 Deemen17 <https://github.com/Deemen17>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+#define I2C_DRIVER I2CD1
+#define I2C1_SDA_PIN GP18
+#define I2C1_SCL_PIN GP19
+
+#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_SWIRL + 5
+
+#ifdef RGB_MATRIX_ENABLE
+#    define DRIVER_ADDR_1 0b0110000
+#    define DRIVER_COUNT 1
+#    define RGB_MATRIX_LED_COUNT 67
+#    define RGB_DISABLE_AFTER_TIMEOUT 1 // number of ticks to wait until disabling effects  
```

Comment:
uuuuh.  So immediately disable?

Reply:
HI @drashna! I've remove RGB_DISABLE_AFTER_TIMEOUT to to avoid conflicts. Please review any corrections if necessary! Thanks!